cohort-cos-cache/pom.xml

```diff
@@ -0,0 +1,117 @@
+<!--  ************************************** -->
+<!--  (C) Copyright IBM Corp. 2021, 2021     -->
+<!--  SPDX-License-Identifier: Apache-2.0    -->
+<!--  ************************************** -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>com.ibm.cohort</groupId>
+		<artifactId>cohort-parent</artifactId>
+		<version>${revision}</version>
+		<relativePath>../cohort-parent</relativePath>
+	</parent>
+
+	<artifactId>cohort-cos-cache</artifactId>
+	<packaging>jar</packaging>
+
+	<scm>
+		<url>https://github.com/Alvearie/quality-measure-and-cohort-service</url>
+		<connection>scm:git:git@github.com:Alvearie/quality-measure-and-cohort-service.git</connection>
+		<developerConnection>scm:git:git@github.com:Alvearie/quality-measure-and-cohort-service.git</developerConnection>
+	</scm>
+
+	<dependencies>
+		<dependency>
+			<groupId>com.ibm.cos</groupId>
+			<artifactId>ibm-cos-java-sdk-core</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>com.ibm.cos</groupId>
+			<artifactId>ibm-cos-java-sdk-s3</artifactId>
+		</dependency>
```

Comment:
For my `cos-playground` project, I was able to get away with just a single dependency on `ibm-cos-java-sdk`.
In fact, you add that dependency to the BOM, but don't explicitly depend on it anywhere.

Does that pull in a bunch of extra dependencies you wanted to avoid?

Reply:
```bash
Warning:  Used undeclared dependencies found:
1233
Warning:     com.ibm.cos:ibm-cos-java-sdk-s3:jar:2.9.0:compile
1234
Warning:  Unused declared dependencies found:
1235
Warning:     com.ibm.cos:ibm-cos-java-sdk:jar:2.9.0:compile
```

is the [error](https://github.com/Alvearie/quality-measure-and-cohort-service/runs/2291182385) I get if I do what you're suggesting. Do you know how to resolve that?

Reply:
Ah, I forgot about the unused dependency checker.  Yea do whatever you need to satisfy the checker.  Sorry for the confusion.